core/src/test/java/org/elasticsearch/common/lucene/LuceneTests.java

```diff
@@ -405,6 +406,7 @@ public void testAsSequentialAccessBits() throws Exception {
      */
     public void testMMapHackSupported() throws Exception {
         // add assume's here if needed for certain platforms, but we should know if it does not work.
+        assumeTrue("Lucene 6.3.0 doesn't support unmap with Java 9", Constants.JVM_SPEC_VERSION.startsWith("1.9") == false);
```

Comment:
We use `assumeFalse(Constants.JRE_IS_MINIMUM_JAVA9);` elsewhere`, I don't think this is right since the version string no longer has the leading `1.`.